Token_Contracts/contracts/StandardToken.sol

```diff
@@ -16,25 +16,23 @@ contract StandardToken is Token {
         //Default assumes totalSupply can't be over max (2^256 - 1).
         //If your token leaves out totalSupply and can issue more tokens as time goes on, you need to check if it doesn't wrap.
         //Replace the if with this one instead.
-        //if (balances[msg.sender] >= _value && balances[_to] + _value > balances[_to]) {
-        if (balances[msg.sender] >= _value) {
-            balances[msg.sender] -= _value;
-            balances[_to] += _value;
-            Transfer(msg.sender, _to, _value);
-            return true;
-        } else { return false; }
+        //require(balances[msg.sender] >= _value && balances[_to] + _value > balances[_to]);
+        require(balances[msg.sender] >= _value);
```

Comment:
I don't think we need it. EIP610 says we return true on success and throw on failure. The `else` block was returning false, but if we are going to throw on failure it's impossible to return false.